src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {

```

Comment:
Util class should never be instantiated

make private constructor to ensure it